.yamllint

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Do we have a standard template? I can't find one. Where did you pick this one?

Reply:
Discussing it @ https://github.com/dnsimple/dnsimple-engineering/pull/248